Sources/MongoSwift/MongoClient.swift

```diff
@@ -14,24 +14,32 @@ public struct ClientOptions: CodingStrategyProvider {
     /// be used.
     public let readConcern: ReadConcern?
 
-    /// Specifies a ReadPreference to use for the client.
-    public let readPreference: ReadPreference?
-
     /// Specifies a WriteConcern to use for the client. If one is not specified, the server's default write concern
     /// will be used.
     public let writeConcern: WriteConcern?
 
+    // swiftlint:disable redundant_optional_initialization
+
+    /// Specifies a ReadPreference to use for the client.
+    public var readPreference: ReadPreference? = nil
```

Comment:
`ReadPreference` isn't `Decodable`, so I needed to provide a default value for it here. swiftlint doesn't detect the semantic value, so I have to disable the rule. The coding strategies are swift specific, so they also need default values.